docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,303 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0790e633-dd4b-4217-b32f-3e33ed56e739",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels=(df[class_names].to_numpy())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2227068f-4382-4e66-b2e4-4fc037223128",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def convert_multilabel_y_to_list(y):\n",
+    "    yl = list()\n",
+    "    for i in range(y.shape[0]):\n",
+    "        yl.append(np.where(y[i,:])[0].tolist())\n",
+    "    return yl    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29ab0d31-5002-4935-8f96-3847edbc172c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multi_labels = convert_multilabel_y_to_list(labels)\n",
+    "\n",
+    "pred_labels = convert_multilabel_y_to_list(np.array((pred_probs>0.5),np.int32))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a973506-c30e-4409-ac65-495537d13730",
+   "metadata": {},
+   "source": [
+    "## 3. Use cleanlab to find label issues \n",
+    "\n",
+    "Based on the given labels and out-of-sample predicted probabilities, cleanlab can quickly help us identify label issues in our dataset. Here we request that the indices of the identified label issues be sorted by cleanlab’s normalized margin score, which measures the quality of each given label via the probability assigned to it in our model’s prediction. The returned `issues` are a list of indices `i`, which corresponds to a label issue at the `i`-th image sample in the dataset. These are the labels cleanlab thinks may be badly labeled in your dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55b5c376-2c02-43d0-a5fc-ceef6f80b0e2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "issues = find_label_issues(pred_probs=pred_probs,labels=multi_labels,\n",
+    "                           multi_label=True,return_indices_ranked_by='normalized_margin')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5a41c12-3002-4691-930f-dac5214443bf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def rep_dl(s):\n",
+    "    return os.getcwd()+'/archive/img_align_celeba/img_align_celeba/'+s\n",
+    "\n",
+    "data2 = np.array([rep_dl(i) for i in df.index])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2b71fa0-ede8-4d16-8f13-da8d98acc90e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_label_name(ls):\n",
+    "    return [class_names[i] for i in ls]\n",
+    "\n",
+    "pred_label_names = [get_label_name(i) for i in pred_labels]\n",
+    "label_names = [get_label_name(i) for i in multi_labels]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fffa88f6-84d7-45fe-8214-0e22079a06d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = 10\n",
+    "h = 10\n",
```

Comment:
[nit] try to use more informative variable names

Reply:
Fixed: https://github.com/cleanlab/cleanlab/pull/517/commits/7aa53405e8abc5f3f0aa2f2a3a96f90f21516db1